packages/ubuntu_test/lib/src/integration_test.dart

```diff
@@ -0,0 +1,144 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:path/path.dart' as p;
+import 'package:subiquity_client/subiquity_server.dart';
+import 'package:ubuntu_wizard/widgets.dart';
+
+// ignore_for_file: invalid_use_of_visible_for_testing_member
+
+/// The path to the `.subiquity` directory.
+final String subiquityPath = p.join('subiquity', '.subiquity');
+
+/// ".subiquity/run/subiquity/states/"
+final String statePath = p.join(subiquityPath, 'run', 'subiquity', 'states');
+
+/// e.g. ".subiquity/run/subiquity/states/Locale"
+String stateFile(String fileName) => p.join(statePath, p.basename(fileName));
+
+/// ".subiquity/etc/"
+final String configPath = p.join(subiquityPath, 'etc');
+
+/// e.g. ".subiquity/etc/wsl.conf"
+String configFile(String fileName) => p.join(configPath, p.basename(fileName));
+
+/// "integration_test/goldens/"
+final String goldenPath = p.join('integration_test', 'goldens');
+
+/// e.g. "integration_test/files/basic-setup/wsl.conf"
```

Comment:
The example doesn't look correct, it should be `"integration_test/goldens/wsl.conf"`.